### PR TITLE
Notes feature

### PR DESCRIPTION
**Pull request by katbow**
Add functionality for updating a note on the front end and syncing it with the db

Styles the notes page and minor style changes for other pages.

(this is branched off the errors branch so worth checking that one out before)

### CODE REVIEW

**Comment by rjmk** [comment]
Conflicts again!

File: lib/db/data.js [hunk 1]
```diff
@@ -135,6 +137,17 @@ const db = (connection) => {
     });
   };
 
+  const makeNoteQuery = (connection, item) =>
```
Comment (rjmk) [inline lib/db/data.js]: It seems a bit strange to mix this method with our other methods (e.g. `addSet`), but I think this is neat, so OK

**Comment by rjmk** [comment]
OK

File: lib/db/data.js (hunk 1 above)
Comment (andrewMacmurray) [inline lib/db/data.js]: we were at pains to do this but the `UPDATE` syntax doesn't play nicely with the other methods

**Comment by andrewMacmurray** [comment]
Thanks @rjmk ! Will miss working with you 😭  it's been so good having you on board

**Comment by rjmk** [comment]
Aww, thanks! You guys have been fantastic to work with :)

**Comment by katbow** [comment]
@rjmk it's been fab :tada: ! We both learned so much from you :heart:. Will definitely miss you!

**Pull request merged**